Sources/StreamChatSwiftUI/Utils/MessagePreviewFormatter.swift

```diff
@@ -0,0 +1,90 @@
+//
+// Copyright © 2024 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import SwiftUI
+
+/// A formatter that converts a message to a text preview representation.
+/// By default it is used to show message previews in the Channel List and Thread List.
+struct MessagePreviewFormatter {
+    @Injected(\.chatClient) var chatClient
+
+    public init() {}
```

Comment:
do we need a public init if the struct is internal?

Reply:
Ah yes does not make sense

Reply:
Fixed ✅